sqlx-core/src/postgres/type_info.rs

```diff
@@ -995,20 +995,16 @@ impl PartialEq<PgType> for PgType {
         if let (Some(a), Some(b)) = (self.try_oid(), other.try_oid()) {
             // If there are OIDs available, use OIDs to perform a direct match
             a == b
+        } else if let (PgType::DeclareWithName(_), PgType::DeclareWithOid(_))
+        | (PgType::DeclareWithOid(_), PgType::DeclareWithName(_)) = (self, other)
+        {
```

Comment:
That's kinda gross. How about
```rust
if matches! (
    (self, other),
    (PgType::DeclareWithName(_), PgType::DeclareWithOid(_)) 
        | (PgType::DeclareWithOid(_), PgType::DeclareWithName(_))
) {
```

Does that format better?

Reply:
rustfmt won't touch most macro invocations (maybe just all that don't look like regular function calls). So the only thing it will do to your example is removing the space between `!` and `(`.

Reply:
That looks much better to me, thank you.